src/lib/connectors/magicAuthConnector.ts

```diff
@@ -57,6 +59,10 @@ export class MagicAuthConnector extends MagicConnector {
     this.redirectUrl = config.options.redirectURI;
   }
 
+  setEmail(email: string) {
```

Comment:
wasn't the plan to add this to the connect call? Or was my hunch that it didn't compile correct?

Reply:
No, since the connect call is never called directly :( 
It's called from the `useConnect` wagmi library, and we can't pass any additional info.